### PR TITLE
Fix github url strings (org edx -> openedx)

### PR DESCRIPTION
**Pull request by sarina**
## This PR was autogenerated

This pr replaces the old GitHub organization, github.com/edx, with the new GitHub organization, github.com/openedx.

Tagging @openedx/tcril-engineering for review, but others are welcome to provide review.

Ref: https://github.com/openedx/tcril-engineering/issues/42

### CODE REVIEW

File: pylintrc [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # ** DO NOT EDIT THIS FILE **
 # ***************************
 #
-# This file was generated by edx-lint: https://github.com/edx/edx-lint
+# This file was generated by edx-lint: https://github.com/openedx/edx-lint
```
Comment (timmc-edx) [inline pylintrc:5]: I think this file needs to be regenerated rather than edited directly.

Comment (sarina) [reply pylintrc:5]: @timmc-edx yep! A few people called this out. Could you re-review now that I've run `edx_lint`?

File: pylintrc_backup [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # ** DO NOT EDIT THIS FILE **
 # ***************************
 #
-# This file was generated by edx-lint: https://github.com/edx/edx-lint
```
Comment (timmc-edx) [inline pylintrc_backup:5]: Hmm, we should just delete this file, it shouldn't be here. I can take care of that in a separate PR, though.

**Review by timmc-edx — APPROVED** [review]

File: pylintrc_backup (hunk 2 above)
Comment (sarina) [reply pylintrc_backup:5]: Hmm. In all my test repos this file never showed up, but it did in a number of other repos. I wonder if it's a `.gitignore` thing. I can remove it.

Comment (sarina) [reply pylintrc_backup:5]: Ah, got it. Thanks!

**Pull request merged**